build/npm/refresh-langpack-extension.js

```diff
@@ -0,0 +1,187 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+let locFunc = require("../lib/locFunc");
+let fs = require("fs");
+let path = require("path");
+
+let gulp = require('gulp');
+let vfs = require("vinyl-fs");
+let rimraf = require('rimraf');
+let minimist = require('minimist');
+
+const textFields = {
+	"nameText": 'ads',
+	"displayNameText": 'Azure Data Studio',
+	"publisherText": 'Microsoft',
+	"licenseText": 'SEE SOURCE EULA LICENSE IN LICENSE.txt',
+	"updateText": 'cd ../vscode && npm run update-localization-extension '
+}
+
+//ADS language pack folder length
```

Comment:
Where is this length from? The comment should explain why this value is what it is - the name already tells us that it's the "language pack folder length" so isn't really providing anything useful.

Reply:
This length is to refer to the folder section reading "ads-language-pack-", its used for isolating the ID at the end of the folder.